Sources/UIGestureRecognizer.swift

```diff
@@ -78,24 +80,18 @@ private extension UIGestureRecognizer {
     private func cancelOtherGestureRecognizersThatShouldNotRecognizeSimultaneously() {
         guard let touch = self.trackedTouch else { return }
 
-        let otherRecognizersThatHaveBeganToRecogize = touch.gestureRecognizers.filter {
-            $0 != self && $0.state == .began
+        let otherRecognizersWhichMayRecognize = touch.gestureRecognizers.filter {
+            $0 != self && ($0.state == .began || $0.state == .possible)
         }
 
-        otherRecognizersThatHaveBeganToRecogize.forEach {
```

Comment:
the older terminology still makes sense to me logically – it seems correct to me